app/model/SubscriptionData.scala

```diff
@@ -2,8 +2,10 @@ package model
 
 import com.gu.identity.play.IdUser
 
-case class PaymentData(account: String, sortCode1: String, sortCode2: String, sortCode3: String, holder: String) {
-  val sortCode = s"$sortCode1$sortCode2$sortCode3"
+case class PaymentData(account: String, sortCodeValue: String, holder: String) {
+  val sortCode = sortCodeValue
```

Comment:
Assuming what you want to do is to strip everything but the sort code digits, a slightly better way of writing this would be:

```scala
sortCode.filter(_.isDigit)
```